charts/bulk-scan-orchestrator/values.preview.template.yaml

```diff
@@ -10,6 +10,7 @@ java:
     CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
     DOCUMENT_MANAGEMENT_URL: "http://dm-store-aat.service.core-compute-aat.internal"
     TRANSFORMATION_URL_BULKSCAN: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
+    UPDATE_URL_BULKSCAN: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
```

Comment:
Consistent with `TRANSFORMATION_URL_...`, so this PR is good, but I think we will need another PR to change the naming of these environment variables to sth like `CASE_TRANSFORMATION_URL_{service}` and `CASE_UPDATE_URL_{service}` to make it clearer what they're about.